yacc/wstr.c

```diff
@@ -22,8 +22,8 @@ static int windows_unicode_enabled = WINDOWS_UNICODE;
 static int windows_unicode_strict = 1;
 
 /* Adapted from runtime/win32.c */
-int win_wide_char_to_multi_byte(const wchar_t *s, int slen,
-                                char *out, int outlen)
+int caml_win32_wide_char_to_multi_byte(const wchar_t *s, int slen,
+                                       char *out, int outlen)
```

Comment:
Do we really want to touch `ocamlyacc`? This is a standalone program that doesn't export any libraries.

Reply:
The only reason was to keep it "in sync" with the function it lifts out of `runtime/win32.c` - I don't have a strong opinion either way 🙂 I would have changed it because grep would have matched the original name.

Reply:
Ok, let's keep this change for consistency.